### ISSUE TITLE
Fixup aria role for skip to main content.

### ISSUE THREAD

**Issue by dj2** [cds-snc/digital-canada-ca#2129]
The skip to main content needs accessibility updates. The UL has the role `navigation` which needs to be removed. This should fixup various violations as these appear on lots of pages.  Removing the navigation role should fixup a `minor` and `serious` violation on each page.
